tests/phpunit.xml

```diff
@@ -1,5 +1,10 @@
-<phpunit bootstrap="bootstrap.php"
-         color="true">
+<phpunit
+    bootstrap="bootstrap.php"
+    color="true"
+    convertErrorsToExceptions   = "true"
```

Comment:
Why not the same spaces as color and bootstrap? :D

Reply:
because it's a copy paste? ^^